projects/power_selection/src/resistance_to_temp.c

```diff
@@ -0,0 +1,61 @@
+#include <stdint.h>
+
+#define NUM_OF_RESISTANCES 114
+
+typedef enum {
+  RES_TEMP_TABLE_TEMP = 0,
+  RES_TEMP_TABLE_RES,
+  NUM_RES_TEMP_TABLE_INDICES
+} ResTempTableIndex;
+
+// from
+// http://2avrmz2nom8p47cc28p2743e-wpengine.netdna-ssl.com/wp-content/uploads/2010/11/Thermistor_10K-2.pdf
+const double resistance_temperature_in_C[NUM_OF_RESISTANCES][2] = {
```

Comment:
can you use the enum value instead of 2 here? also please make this `static` and prefix the name to `s_`